src/components/card/card-row/card-row.style.ts

```diff
@@ -0,0 +1,35 @@
+import styled, { css } from "styled-components";
+import { padding, PaddingProps } from "styled-system";
+import baseTheme, { ThemeObject } from "../../../style/themes/base";
+import { CardSpacing } from "../card.config";
+
+export interface StyledCardRowProps extends PaddingProps {
+  /**
+   * Spacing prop is set in Card and defines the padding for the CardRow (the first CardRow has no padding by default).
+   * For more granular control of CardRow padding these can be over-ridden by Padding props from styled-system.
+   */
+  spacing: CardSpacing;
+  /** @ignore @private */
+  theme?: ThemeObject;
```

Comment:
The `theme` prop does not necessarily have to be there, since the object is not used in the component's code. However, we need to set the theme in `defaultProps` otherwise styled system will use its default spacing scale when calculating custom padding, whereas we want to use our own using design tokens.